java/src/jmri/jmrix/can/cbus/swing/modules/CbusConfigPaneProvider.java

```diff
@@ -111,10 +97,15 @@ final static public CbusConfigPaneProvider getProviderByNode(CbusNode node) {
         CbusConfigPaneProvider p = instanceMap.get(node.getName());
         if (p != null) {
             return p;
-        } else {
-            log.debug("node {} NN {} name {} gets unknown provider", node, node.getNodeNumber(), node.getName());
-            return new UnknownPaneProvider();
+        } else if (node.getResyncName() != null) {
+            // Get the saved name during a resync
+            p = instanceMap.get(node.getResyncName());
+            if (p != null) {
+                return p;
+            }
         }
+        log.info("node gets unknown provider: {}", node);
```

Comment:
This message went from being `debug` to `info`.  If that was intentional, OK, but maybe it means it should be a `warn`?